### DISCUSSION TITLE
SQLSTATE[HY000]: General error: 1364 Field 'locked' doesn't have a default value.

### DISCUSSION THREAD

**Discussion by uicabpatweyler** [spatie/laravel-settings#221] [General]
I have created my Filament page settings, the settings table in my database has been created successfully, I have added the initial data.
 
I can correctly view the data that is initially created during the migration, but when I want to update it, it shows me the error.

SQLSTATE[HY000]: General error: 1364 Field 'locked' doesn't have a default value.

Any idea why this error happens?

**Comment by filip-paral** [comment]
See [Issue](https://github.com/spatie/laravel-settings/issues/217)
